var/spack/repos/builtin/packages/mxml/package.py

```diff
@@ -31,8 +31,7 @@ class Mxml(AutotoolsPackage):
     non-standard libraries.
     """
 
-    homepage = "http://www.msweet.org"
-    url = "http://www.msweet.org/files/project3/mxml-2.9.tar.gz"
```

Comment:
I added a default URL. To make it work, I also had to amend the `parse_name_offset` function, otherwise `spack versions mxml` fails with `Couldn't parse package name`.